src/core/import-html/html-to-json.ts

```diff
@@ -7,6 +7,8 @@ import { STYLES_KEY } from "../constants/CONTROLS";
 import { omit } from "lodash";
 import { getVideoURLFromHTML, hasVideoEmbed } from "./import-video.ts";
 
+const DATA_CHAI_REPLACE_PREFIX = "data-chai-replace-";
```

Comment:
@RafaelGCruvinel I think we should use a smaller attr name for quickly adding attrs while developing html.

Reply:
yes.. the only point is to use "data-*".. do you have any recomendation? I think a little about it..

Reply:
i think simply `data-chai-{key}` should be good.